woodwork/type_sys/type_system.py

```diff
@@ -91,6 +91,12 @@
     (Integer, Age),
     (IntegerNullable, AgeNullable),
     (IntegerNullable, Integer),
+    (Unknown, EmailAddress),
+    (Unknown, IPAddress),
+    (Unknown, PhoneNumber),
```

Comment:
should any of these have categorical parents?

Reply:
My idea here was that these values don't necessarily have categorical-like values, since there's an infinite number of possible values and when they appear in datasets, they have the potential to be all-unique. This is different from when `CountryCode` or `CurrencyCode` appear, since there's a limited number of values these could be. That's why I decided to opt for `Unknown` as the parent, but let me know what you think!

Reply:
Hmm, yeah I mean it definitely depends on the dataset honestly. If there are few enough distinct values, it probably should be categorical, and if not, it should be Unknown. I don't suppose there's any way to represent something like that currently is there?

Reply:
Yep, currently no, but we only use the parent-child relationships defined to determine which value we prefer to default to. If a dataset matches with our Categorical inference and URL inference, it'll default to URL now. I believe if it matches with Categorical and Unknown, it will choose the first match, which will be Categorical.